v2/constants.go

```diff
@@ -5,6 +5,11 @@
 
 package v2
 
+// Miscellaneous constants
+const (
+	CorrelationHeader = "X-Correlation-ID" // Sets the key of the Correlation ID HTTP header
```

Comment:
The value defined in V1 and V2 are the same.  However, it will be easier to put everything under v2 package right now, and we can be remove the v1 clients package once we decide to clean up the v1 code.

Reply:
My concern is that this constant is used in more places than REST APIs such as log message in App Service message bug processing. I.e Correlation Id is not specific to REST. 

IMO, the current location is OK and independent of the REST APIs

Reply:
no problem, @FelixTing please remove this constant and use the one in clients package

Reply:
no problem, it's done